gcn/notices/core/Reporter.schema.json

```diff
@@ -27,6 +27,12 @@
       "type": "string",
       "default": "energy"
     },
+    "spectral_center": {
+      "enum": ["energy", "wavelength", "frequency"],
+      "description": "Central value of the observing band",
+      "type": "number",
+      "default": "energy"
+    },
     "units": {
```

Comment:
We should specify what these units are for.  When flattened, it'll just be units and we should disambiguate.